examples/edge/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "react-engine-example-edge",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run browserify && node index.js",
+    "browserify": "browserify -t babelify -t require-globify public/index.js -o public/bundle.js"
```

Comment:
I am using babelify instead of reactify, since even Facebook uses Babel instead of their own react-tools nowadays.